fixes #80, #81

ogobrecht opened the pull request:
Hi again,

here the fixes for the two issues opened by me. Feel free to revoke the pull request and let me know what is missing or wrong.

Comment on the syntax highlighting: I learned a lot. Instead of adding one new entry like you mentioned I had to add multiple ones - otherwise it would not work because of mixed start and end characters (regex "or"). Hope this will not slow down the parser.

Comment on the symbol recognition: Beneath the four regexes for the brackets I tried to add one generic one with a back reference like so: `/q'(.)[\s\S]*?\1'/gi`. This is working when I try it on [regexr.com/4g99i](https://regexr.com/4g99i) (I saved it for you) but unfortunately not when I use it in the regex parser for the symbol recognition. I have no idea why this is not working there. So I had also to use here the same approach as for the syntax highlighting and provide for each needed character an own regex expression. Currently I support these characters: ```() [] {} <> | ! # ` ^```.

I added also a file `main/test/sql/ogobrecht_alternativeQuoting.sql`. Please let me know, if I should integrate this test file in the existing package file `main/test/sql/xyz_myPackage2.pkb`.

Best regards
Ottmar

ogobrecht committed:
fixes #80, #81

jeanremacle commented:
Hello,

I was just looking into the same fix. 
Are there any chances that this would be applied in a near future?

Regards,
Jean

ogobrecht committed:
update npm module vscode to 1.1.37

- needed this to be able to run npm install without errors
- vscode is deprecated
- 1.137 seems to be the last version
- `npm install` was now successful
- `vsce package` was also successful

ogobrecht committed:
Create plsql-language-1.8.2111.vsix

ogobrecht committed:
Update .gitignore

ogobrecht committed:
add tilde as an option to alternative quote strings
